types/react-addons-update/index.d.ts

```diff
@@ -6,7 +6,7 @@
 
 import * as React from 'react';
 
-export = React.__Addons.update;
+export default React.__Addons.update;
```

Comment:
It seems like the [underlying library](https://unpkg.com/react-addons-update@15.6.2/index.js) is not using `export default`, so this .d.ts file should not either. Instead, you should continue to use `export =`. Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.